### DISCUSSION TITLE
Best way to embed policies with JMESPath into Helm templates

### DISCUSSION THREAD

**Discussion by cybra-northco** [kyverno/policies#1030] [Q&A] [answered]
Looking at the https://github.com/kyverno/kyverno/blob/main/charts/kyverno-policies/templates/restricted/disallow-capabilities-strict.yaml line 81:
```
value: "{{`{{`}} element.securityContext.capabilities.drop[] || `[]` {{`}}`}}"
```

Wouldn't it be better to write it like this:
```
value: {{ `"{{ element.securityContext.capabilities.drop[] || \"[]\" }}"` }}
```

**Comment by chipzoller** [comment]
In JMESPath, the notation needs to be:

```
`[]`
```

The backticks are needed to denote an object.

**Reply by cybra-northco** [reply]
@chipzoller thank you. What is the outcome of using `\"[]\"`?
In our cluster, kyverno (1.11) doesn't give any warnings when creating a policy manifest with "[]".

**Reply by chipzoller** [reply, marked as answer]
Functionally in this case it probably doesn't matter but in other cases it does so we're trying to be technically correct according to the specifications.
